P0009/P0009.bs

```diff
@@ -1753,16 +1853,22 @@ a contiguous set of objects of type *ElementType* are accessed.
 <br/>
 <b>26.7.�.1 Accessor policy requirements [mdspan.accessor.reqs]</b>
 
-An accessor policy defines an accessor type and operations for applying
-subscripting-like [expr.sub],
-conversion to pointer [conv.array],
-and offset operations to an accessor.
-The subscript operation returns an object which provides access to the indexed element,
-this return type may not be `T&`.
-The offset operation returns an accessor for the contiguous subset of elements beginning at the offste value,
-this return type may not be the same accessor type..
-An accessor constructor may impose additional restrictions on the 
-contiguous set of objects; e.g., restrict `T` to be trivially copyable or have a wider alignment.
+An accessor policy defines
+a type through which a single element is accessed,
+a type through which the contiguous set of elements is accessed, 
```

Comment:
What does "through" mean?

Reply:
"defines a generalization of pointer, which is a handle to the contiguous set of elements."  (relax contiguity later, in a later paper, e.g., for segmented allocations)

Reply:
"defines a reference, which is a handle to a single element"